src/features/lightspeed/lightspeedUser.ts

```diff
@@ -299,6 +348,25 @@ export class LightspeedUser {
 
     return this._userDetails;
   }
+  public async getMarkdownLightspeedUserDetails(
+    createIfNone: boolean,
+    useProviderType: AuthProviderType | undefined = undefined,
+  ) {
+    // Ensure we don't try to get a lightspeed auth session when the provider is not initialized
+    if (!this._settingsManager.settings.lightSpeedService.enabled) {
+      return undefined;
```

Comment:
To be fair, if we reach this case, it means we are already in serious trouble because half of the extension is not properly initialized.